packages/scandipwa/src/component/WishlistItem/WishlistItem.component.js

```diff
@@ -258,7 +259,7 @@ export class WishlistItem extends PureComponent {
 
         return (
             <div block="WishlistItemOptions" elem="List">
-                { options.map(({ value }) => value).join(', ') }
+                { options.map(({ value }) => <Html content={ `${ value }<br>` } />) }
```

Comment:
Using the `<br>` tag is usually a bad practice.
Maybe it's better to wrap it with the `<div>`?